torch_geometric/nn/conv/message_passing.py

```diff
@@ -630,6 +629,8 @@ def jittable(self, typing: Optional[str] = None):
                 instance with :meth:`forward` types based on :obj:`typing`,
                 *e.g.*: :obj:`"(Tensor, Optional[Tensor]) -> Tensor"`.
         """
+        from jinja2 import Template
```

Comment:
Do you think it might be worth having a warning if jinja2 is not installed when importing message_passing, otherwise when the import error comes later it can be a bit confusing.

Reply:
Done.